spec/support/detect_rails_version.rb

```diff
@@ -1,19 +1,5 @@
-# Detects the current version of Rails that is being used
-#
-#
-RAILS_VERSION_FILE ||= File.expand_path("../../../.rails-version", __FILE__)
-
-unless defined? TRAVIS_CONFIG
-  require 'yaml'
-  filename = File.expand_path("../../../.travis.yml", __FILE__)
-  TRAVIS_CONFIG = YAML.load_file filename
-  TRAVIS_RAILS_VERSIONS = TRAVIS_CONFIG['env']['matrix'].grep(/RAILS=(.*)/){ $1 }
-end
-
-DEFAULT_RAILS_VERSION ||= TRAVIS_RAILS_VERSIONS.last
-
 def detect_rails_version
-  version = version_from_file || ENV['RAILS'] || DEFAULT_RAILS_VERSION
+  version = ENV['BUNDLE_GEMFILE'].match(/rails_(.*).gemfile/)[1]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.